src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -15,6 +15,10 @@
 use Symfony\Component\HttpKernel\DataCollector\LateDataCollectorInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Twig\Markup;
+use Twig\Profiler\Dumper\HtmlDumper;
+use Twig\Profiler\Profile;
+use Twig\Source;
```

Comment:
@nicolas-grekas Not currently, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2814.